common/test/acceptance/pages/studio/index.py

```diff
@@ -290,6 +290,27 @@ def language_selector(self):
         )
         return self.q(css='#settings-language-value')
 
+    @property
+    def error_text_length(self):
```

Comment:
Could this be renamed to "course_creation_error_message" or such?  From the name, I thought this would return the number of characters in the error message.